libdrgn/debug_info.c

```diff
@@ -2433,6 +2433,21 @@ drgn_type_from_dwarf_internal(struct drgn_debug_info *dbinfo,
 					 "maximum DWARF type parsing depth exceeded");
 	}
 
+	/* If we get a declaration, try to follow it to a specification */
+	bool decl;
+	if (!dwarf_flag(die, DW_AT_declaration, &decl) && decl) {
+		size_t die_addr;
```

Comment:
This should be `uintptr_t` (it's probably the same as `size_t` on every sane platform, but better to be consistent with how `drgn_dwarf_index_find_definition()` is declared).